src/taipy/core/_manager/_manager.py

```diff
@@ -97,3 +97,10 @@ def _delete_entities_of_multiple_types(cls, _entity_ids: _EntityIds):
         _TaskManagerFactory._build_manager()._delete_many(_entity_ids.task_ids)
         _JobManagerFactory._build_manager()._delete_many(_entity_ids.job_ids)
         _DataManagerFactory._build_manager()._delete_many(_entity_ids.data_node_ids)
+
+    @classmethod
+    def _export(cls, id, **kwargs):
```

Comment:
```suggestion
    def _export(cls, entity: Union[Entity, str], folder_path: str):
```

Is there another format for exporting entities or we will only export them as JSON? If we'll only be exporting them as JSON, I think folder_path is better choice than kwargs

Reply:
The export format will always be JSON. I do agree that the folder_path is a better implementation. It makes the function usage self explanatory.